client/x/evmengine/types/expected_keepers.go

```diff
@@ -19,11 +19,14 @@ type AccountKeeper interface {
 type EvmStakingKeeper interface {
 	ProcessDeposit(ctx context.Context, ev *bindings.IPTokenStakingDeposit) error
 	ProcessWithdraw(ctx context.Context, ev *bindings.IPTokenStakingWithdraw) error
```

Comment:
Could you delete these two as well? As I remember, these two are not used in evmengine keeper.